src_bicep/predeploy/main.bicep

```diff
@@ -0,0 +1,44 @@
+var settings = loadJsonContent('main.parameters.json')
+param location string = resourceGroup().location
+
+resource identity 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31' existing = {
+  name: settings.identityName
+}
+
+resource galleries 'Microsoft.Compute/galleries@2022-03-03' = {
+  name: settings.galleryName
+  location: location
+  properties: {}
+}
+
+// Role Assignment - Contributor
+resource roleAssignmentsContributor 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
+  name: guid(settings.galleryName, 'Contributor')
+  properties: {
+    roleDefinitionId: subscriptionResourceId('Microsoft.Authorization/roleDefinitions', 'b24988ac-6180-42a0-ab88-20f7382dd24c')
```

Comment:
Can we change `b24988ac-6180-42a0-ab88-20f7382dd24c` to it's name which is [Contributor](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#all).

Reply:
Didn't it work?